pages/finance.tsx

```diff
@@ -0,0 +1,22 @@
+import { useUser } from '@auth0/nextjs-auth0'
+import React from 'react'
+
+import BaseLayout from '../components/common/BaseLayout'
+import FinanceDateHeader from '../components/finance/FinanceDateHeader'
+
+function Finance() {
+  const { user, isLoading, error } = useUser()
+
+  if (isLoading) return <div>Loading...</div>
+  if (error) return <div>{error.message}</div>
+
+  // TODO: add useOrientation using custom hook.
+  const left = -5
```

Comment:
need to calculate this using page width so that we can use the entire real estate

Reply:
at present using startDate and endDate instead. Would pick in next PR for responsiveness